### PR TITLE
Exclude more generated sources and plugin instrumentation definition …

### PR DESCRIPTION
**Pull request by kezhenxu94**
…classes

Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
- [x] Test

- Related issues

#2438

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -550,6 +550,17 @@
                             <exclude>org/apache/skywalking/oap/server/exporter/grpc/*.class</exclude>
                             <exclude>org/apache/skywalking/oap/server/configuration/service/*.class</exclude>
                             <exclude>grpc/health/v1/*.class</exclude>
+
+                            <exclude>org/apache/skywalking/apm/toolkit/**/*Activation.class</exclude>
+                            <exclude>org/apache/skywalking/apm/plugin/**/*Instrumentation.class</exclude>
+                            <exclude>org/apache/skywalking/apm/plugin/**/*Instrumentation$*.class</exclude>
```
Comment (kezhenxu94) [inline pom.xml]: Agent plugin instrumentation definition classes

File: pom.xml [hunk 2]
```diff
@@ -550,6 +550,17 @@
...
+
+                            <exclude>com/google/api/**/*.class</exclude>
+                            <exclude>io/envoyproxy//**/*.class</exclude>
+                            <exclude>io/istio/**/*.class</exclude>
+                            <exclude>io/prometheus/**/*.class</exclude>
+                            <exclude>io/jaegertracing/**/*.class</exclude>
+                            <exclude>com/lyft/**/*.class</exclude>
```
Comment (kezhenxu94) [inline pom.xml]: Receiver proto generated classes

**Comment by wu-sheng** [comment]
image

The generated codes still counted.

**Pull request merged**